docs/ionic/authentication.md

```diff
@@ -0,0 +1,94 @@
+# STEP 1: install this plugin: 
+```
+1) ionic cordova plugin add cordova-universal-links-plugin
+2) ionic cordova plugin add cordova-plugin-buildinfo
+3) ionic cordova plugin add cordova-plugin-browsertab
+4) ionic cordova plugin add cordova-plugin-inappbrowser
+5) ionic cordova plugin add cordova-plugin-customurlscheme (for ios)
+```
+
+# STEP 2: Add Firebase to your Ionic
+
+  STEP 2.1:  [setup firebase to angular project] 
```

Comment:
Instead of sub numbering the steps just place the action in bold.

**Setup Firebase to the Angular Project**